### ISSUE TITLE
Vulkan on Intel doesn't work with Mesa >= 21.3 (missing shaderSharedInt64Atomics)

### ISSUE THREAD

**Issue by jbeich** [yuzu-emu/yuzu#7116]
Regressed by https://github.com/mesa3d/mesa/commit/2e5718c957415. `shaderSharedInt64Atomics` [is disabled](https://github.com/mesa3d/mesa/blob/b40d070ab9aa/src/intel/vulkan/anv_device.c#L1353) in anv, [now aborting](https://github.com/mesa3d/mesa/blob/b40d070ab9aa/src/vulkan/util/vk_physical_device_features.py#L158) in Yuzu. Tested on Skylake with Super Mario Odyssey.
```
$ vulkaninfo | fgrep atomic_int64
        VK_KHR_shader_atomic_int64                  : extension revision 1

$ yuzu
[...]
MESA-INTEL: error: mesa-git-b40d070ab9a/src/intel/vulkan/anv_device.c:2924: VK_ERROR_FEATURE_NOT_PRESENT
Render.Vulkan <Error> video_core/renderer_vulkan/renderer_vulkan.cpp:RendererVulkan:116: Vulkan initialization failed with error: VK_ERROR_FEATURE_NOT_PRESENT
HW.GPU <Error> :CreateGPU:51: Failed to initialize GPU: Vulkan initialization error VK_ERROR_FEATURE_NOT_PRESENT
Core <Critical> core/core.cpp:Load:235: Failed to initialize system (Error 5)!
```
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7270727/yuzu_log.txt)

**Comment by Joey11904** [comment]
Not sure if its related but I also have the same issue:
```
$ ./yuzu-20211231-6591103c2.AppImage 
QPixmap::scaled: Pixmap is a null pixmap
QPixmap::scaled: Pixmap is a null pixmap
QPixmap::scaled: Pixmap is a null pixmap
QPixmap::scaled: Pixmap is a null pixmap
QPixmap::scaled: Pixmap is a null pixmap
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

Unhandled SIGSEGV at rip 0x00555bb3c6c551
Segmentation fault (core dumped)
```

here's the log:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7797583/yuzu_log.txt)

**Comment by DadSchoorse** [comment]
This should be fixed by #7747, would be nice if you can confirm that.

**Comment by jbeich** [comment]
I confirm, works fine after applying #7747. Tested on Mesa 22.0.0-devel (git-96cc3007461 from 2022-01-19).
